Add validation `id` output

forevermatt opened the pull request:
### Added
- Add an output that might help us wait for certificate validation
  * Attribute documentation is [here](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/acm_certificate_validation#id).

forevermatt committed:
Add an output that might help us wait for certificate validation

Attribute documentation is here:
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/acm_certificate_validation#id

forevermatt committed:
Fix reference to resource that might not exist (since it has a `count`)

forevermatt committed:
If the validation resource won't exist, don't try to get its `id`

Baggerone commented on outputs.tf:
```diff
@@ -3,3 +3,8 @@ output "certificate_arn" {
+
+output "validation_id" {
+  description = "The time at which the certificate was issued"
+  value       = var.create_dns_validation ? one(aws_acm_certificate_validation.this).id : null
```
I think you need to include `[*]` after `.this`.

Do you know whether it works without it?

forevermatt replied:
Updated. Thanks.

forevermatt committed:
Use a simpler syntax of `[*]` rather than ternary operator

Taken from the example in the documentation for the `one()` function:
https://developer.hashicorp.com/terraform/language/functions/one
